apps/web/src/app/api/projects/[projectId]/commits/[commitUuid]/documents/route.ts

```diff
@@ -23,7 +23,7 @@ export async function GET(
     const commit = await commitsScope
       .getCommitByUuid({ uuid: commitUuid, project })
       .then((r) => r.unwrap())
-    const documents = await scope.getDocumentsAtCommit(commit)
+    const documents = await scope.getDocumentsAtCommit({ commit })
```

Comment:
We have discussion offline about using or not positional arguments. The conclusion we arrive is that we hate it. No, I'm joking (kind of). 

The conclusion was that we're going to have methods with positional arguments when we're sure this method will never have more arguments. This is an exception because I did this refactor before arriving to that conclusion and I'm sick of having this PR open. Feel free to change this one if you want.